spec/models/author_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe Author, type: :model do
+  let(:author){
+    FactoryBot.create :author
+  }
+
+  let(:author_1){
+    FactoryBot.create :author
+  }
+
+  let(:author_2){
+    FactoryBot.create :author
+  }
+
+  describe "presence" do
+    it { is_expected.to validate_presence_of(:name) }
+    it { is_expected.to validate_presence_of(:gender) }
+    it { is_expected.to validate_presence_of(:dob) }
+    it { is_expected.to validate_presence_of(:description) }
+  end
+
+  describe "Associations" do
+    it { should have_many(:books).dependent(:destroy) }
+  end
+
+  describe "Validations" do
+    context "when field name" do
```

Comment:
context/it dùng để làm gì

Reply:
- context tạo ra một scope, thường được sử dụng để tạo ra ngữ cảnh (trường hợp) test.
- context hay được sử dụng ở bên trong scope của describe
- it ở trong context hoặc describe để tạo một example